.env.template

```
@@ -1,23 +1,27 @@
 NEXT_PUBLIC_APP_URL=http://localhost:3000
 NEXT_PUBLIC_SUBSCRIPTION_BASE_URL=http://localhost:8001
-NEXT_PUBLIC_HMAC_SECRET_KEY=dummy
 NEXT_PUBLIC_WORKFLOWS_URL=http://localhost:8002
 NEXT_PUBLIC_WORKFLOWS_WEBHOOK_URL=https://e7ae-2405-201-d007-30eb-6751-8af2-6edb-b5a4.ngrok-free.app
+NEXT_PUBLIC_SENTRY_REDIRECT_URI=http://localhost:8001/integrations/sentry/callback
+NEXT_PUBLIC_LINEAR_REDIRECT_URI=http://localhost:8001/integrations/linear/callback
+NEXT_PUBLIC_JIRA_REDIRECT_URI=http://localhost:8001/integrations/jira/callback
+NEXT_PUBLIC_CONFLUENCE_REDIRECT_URI=http://localhost:8001/integrations/confluence/callback
+NEXT_PUBLIC_SLACK_SERVER=https://slack.potpie.ai
 
 
```

dotenv-linter (4.0.0)

[warning] [ExtraBlankLine] Extra blank line detected

(ExtraBlankLine)